test/SpokePool.Admin.ts

```diff
@@ -0,0 +1,80 @@
+import { expect } from "chai";
+import { Contract } from "ethers";
+import { ethers } from "hardhat";
+import { SignerWithAddress } from "./utils";
+import { ZERO_ADDRESS } from "@uma/common";
+import { deploySpokePoolTestHelperContracts } from "./SpokePool.Fixture";
+import { depositDestinationChainId } from "./constants";
+
+let spokePool: Contract, erc20: Contract;
+let admin: SignerWithAddress;
```

Comment:
nit: in the other tests we call this the `owner`. we should be consistant. either we should change this files test or the other one to reflect `admin` over `owner.

Reply:
ok i'm good with owner